getMarks/getMarks.js

```diff
@@ -0,0 +1,68 @@
+const constants = require('../utils/constants')
+
+const getMarks = (req, res) => {
+  const token = req.body.token;
+  const subjectId = req.body.subjectId;
+  const isRightSubjectId = checkSubjectId(subjectId);
+
+  if (!isRightSubjectId) {
+    const responseData = { message: "invalid subject id" };
+    res.status(404).send(responseData);
+  } else {
+    const responseData = getMarksBySubjectId(subjectId, token);
+    res.status(responseData.status).send(responseData.marksData);
+  }
+
+}
+
+function checkSubjectId(id) {
+  if (id > 0 && id <= constants.CONSTANTS.MOCK_SUBJECTS.length) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function getMarksBySubjectId(id, token) {
+  const statusOk = 200;
+  const statusUnauthorized = 401;
+  const response = {
```

Comment:
From design(and performance) perspective, it would be better to check token first to avoid some unnecessary operations if token is wrong.